CHANGELOG.md

```diff
@@ -6,6 +6,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.31.0] - 2024-09-03
+
+### Added
+- Parameter `meta: dict` on `BaseEvent`.
+
+### Changed
+- **BREAKING**: Drivers, Loaders, and Engines will now raises exceptions rather than returning `ErrorArtifacts`.
+- **BREAKING**: Parameter `driver` on `BaseConversationMemory` renamed to `conversation_memory_driver`.
+- **BREAKING**: `BaseConversationMemory.add_to_prompt_stack` now takes a `prompt_driver` parameter.
+- **BREAKING**: `BaseConversationMemoryDriver.load` now returns `tuple[list[Run], Optional[dict]]`.
```

Comment:
have conversation memory iterations always been called "Run"? Could we use "Turn" to avoid confusion with Structure Runs, or is it too late? Also, can we clarify what the Optional dict has in it?

Reply:
Yes, it has always been called this in the framework. This will probably be overhauled soon and we can evaluate new names then.

Reply:
Clarified the optional dict though.